cou/cli.py

```diff
@@ -137,9 +137,8 @@ async def get_upgrade_plan(args: CLIargs) -> None:
     :param args: CLI arguments
     :type args: CLIargs
     """
-    analysis_result, upgrade_plan = await analyze_and_plan(args)
+    _, upgrade_plan = await analyze_and_plan(args)
```

Comment:
I don't think there's any other place needs the analysis_result output from `analyze_and_plan` function. We can change that function to only return `upgrade_plan` and you don't need to ignore an value here.

Reply:
True, I do not think that this is really related with desire goal of this fix, but I removed it.